en/frames_rover/ackermann_rover.md

```diff
@@ -0,0 +1,246 @@
+| <a id="RA_WHEEL_BASE"></a>[RA_WHEEL_BASE](../advanced_config/parameter_reference.md#RA_WHEEL_BASE)       | Wheel-base of the rover which is measured from the back to the front wheel | -    |
+| <a id="RA_MAX_STR_ANG"></a>[RA_MAX_STR_ANG](../advanced_config/parameter_reference.md#RA_MAX_STR_ANG)    | Maximum steering angle of the rover                                        | deg  |
+| <a id="RA_MISS_VEL_DEF"></a>[RA_MISS_VEL_DEF](../advanced_config/parameter_reference.md#RA_MISS_VEL_DEF) | Default velocity the rover will drive during the mission                   | m/s  |
+
+![Geometric parameters](../../assets/airframes/rover/rover_ackermann/geometric_parameters.png)
+
+This is enough to start driving missions, but depending on the rover might not yet lead to satisfactory performance.
+If that is the case further tuning is required which is outlined in [Mission parameters](#mission-parameters).
+
+## Tuning (Advanced)
+
+To get an overview of all parameters that are related to the ackermann rover module navigate to the _Rover Ackermann_ group in the _Parameters_ section of QGroundControl.
+
+### General Parameters
+
+These parameters affect the general behaviour of the rover. This will influence both auto and manual modes.
+
+| Parameter                                                                                       | Description                                | Unit |
+| ----------------------------------------------------------------------------------------------- | ------------------------------------------ | ---- |
+| <a id="RA_MAX_SPEED"></a>[RA_MAX_SPEED](../advanced_config/parameter_reference.md#RA_MAX_SPEED) | Speed the rover drives at maximum throttle | m/s  |
+
+This is used for a feed-forward term on the speed controller in mission mode and necessary for the [acceleration slew rate](#slew-rates).
+
+#### Slew Rates
+
+Slew rates limit how fast the signal that is sent to the motors is allowed to change:
+
+| Parameter                                                                                                | Description                            | Unit  |
+| -------------------------------------------------------------------------------------------------------- | -------------------------------------- | ----- |
+| <a id="RA_MAX_ACCEL"></a>[RA_MAX_ACCEL](../advanced_config/parameter_reference.md#RA_MAX_ACCEL)          | Limit on the acceleration of the rover | m/s^2 |
+| <a id="RA_MAX_STR_RATE"></a>[RA_MAX_STR_RATE](../advanced_config/parameter_reference.md#RA_MAX_STR_RATE) | Limit on the steering rate             | deg/s |
+
+:::warning
+The slew rates are not based on measurements but on assumed linear relation between the throttle input and [RA_MAX_SPEED](#RA_MAX_SPEED) or steering input and [RA_MAX_STR_ANG](#RA_MAX_STR_ANG) respectively.
+Therefore these two parameters have to be set for the slew rates to work!
+:::
+
+### Mission Parameters
```

Comment:
IN my opinion there is enough information here for us to create a mission mode topic for rover and pull all this information into it. Are you OK if I do that?

How much of this would be in common with differential rovers in future?

Reply:
The pure pursuit part is shared, all new USV modules that are currently being worked on use that algorithm.
This also includes the [Pure Pursuit Algorithm](https://github.com/PX4/PX4-user_guide/pull/3265/files#diff-6ad9e9c11f97f5ef09c780ba38fb6c3242db85a2fcdca8e77ed1f28ab387760eR199-R220) section in this PR.
The cornering part is ackermann specific (at least for now).

> IN my opinion there is enough information here for us to create a mission mode topic for rover and pull all this information into it. Are you OK if I do that?

That works for me.